### ISSUE TITLE
Assignment Lecture 4 SIN X function

### ISSUE THREAD

**Issue by sjy268** [goldshakil/ElectricalProgrammingCourse2020#63]
Hello.
For Assignment Lecture 4 SIN X function, I made a program for this assignment but there is an error.
To be specific, the result should be zero when maxloop value is large. But, my result is almost zero, not exactly zero. ex)-1.110223e-16 . Is it okay to regard this result as zero or I have to fix it? And, can you tell me what is the problem?

**Comment by goldshakil** [comment]
Can you share your code with me in inbox? I can't figure out the problem without looking at the code.

**Comment by goldshakil** [comment]
inbox= Icampus msg

**Comment by sjy268** [comment]
> inbox= Icampus msg

there is an error whenever i send an icampus msg. is there another way to send you?
If is it okay send through email, can you tell me your email address.

**Comment by goldshakil** [comment]
here you go : omarshakil100@gmail.com

**Comment by sjy268** [comment]
I sent an email. plz check your email.

**Comment by Gill20000** [comment]
Exactly, I have same error value (-1.110223e-16) whenever I input any degree, if the maxloop is bigger than 6. It looks like the sign of pretty high accuracy, but it is not clear-zero.

I add the captured img.

<img width="326" alt="Error value" src="https://user-images.githubusercontent.com/62375905/78174758-bcd0be00-7494-11ea-892f-38590f57535a.PNG">

**Comment by ysw032** [comment]
I have the same problem... Error between sin(x) and our SIN is zero only if I put the degree as a 60. otherwise, It has tiny error.
is it okay if the error is tiny enough, and the presented values(by sin(x) and SIN) are same???

**Comment by goldshakil** [comment]
I believe you are familiar with how Taylor series works.

This series is supposed to approximate and not get you the exact answer so it makes sense that it is not exactly the same answer.

You can submit your code as it is don't worry.
